rules/cyberark/privilege_escalation_cyberarkpas_error_audit_event_promotion.toml

```
@@ -0,0 +1,61 @@
+[metadata]
+creation_date = "2021/06/23"
+maturity = "production"
+updated_date = "2021/06/23"
+min_stack_version = '7.14.0'
+
+[rule]
+author = ["Elastic"]
+description = "Identifies the occurrence of a CyberArk Privileged Access Security (PAS) error level audit event."
+false_positives = ["To tune this rule, add exceptions to exclude any event.code which should not trigger this rule."]
+from = "now-30m"
+index = ["filebeat-*", "logs-cyberarkpas.audit*"]
+language = "kuery"
+license = "Elastic License v2"
+name = "CyberArk Privileged Access Security Error"
+note = """## Config
+
+The cyberarkpas Fleet integration, Filebeat module, or similarly structured data is required to be compatible with this rule.
```

these names (`cyberarkpas`) aren't made easily visible to the user, I would recommend spelling out the full name.

also, this wording feels a little more natural to me
```suggestion
The cyberarkpas Fleet integration, Filebeat module, or similarly structured data is required for proper function of this rule.
```

---

to me that is _less_ accurate and feels like it implies the logic or operation of the rule changes based on data.

I am not against changing it, but the most important thing we want to imply here is to have the right data, however it gets in

---

++ to spelling out

---

```suggestion
The CyberArk Privileged Access Security (PAS) Fleet integration, Filebeat module, or similarly structured data is required to be compatible with this rule.
```